lib/AuthStateManager.ts

```diff
@@ -131,7 +140,7 @@ class AuthStateManager {
             isPending = shouldEvaluateIsPending();
           }
           let promise = isAuthenticated 
-            ? isAuthenticated(this._sdk)
+            ? isAuthenticated(this._sdk, isPending)
```

Comment:
If we anticipate ever adding more params, I'd prefer to have an object of additional params rather than an increasing list: `isAuthenticated(this._sdk, { isPending })`